README.MD

```diff
@@ -17,7 +17,7 @@
 **NPM**
 
 ```shell
-npm install react-native-joystick react-native-gesture-handler --save
+npm install @korsolutions/react-native-joystick react-native-gesture-handler --save
 ```
 
 **YARN**
```

Comment:
Change the yarn command as well please

Reply:
I've done that. I'm assuming yarn add will work the same way as NPM here.